deployment/arm/deploy.sh

```
@@ -2,17 +2,65 @@
 parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
```

add flag `set -e` to fail early